src/main/java/frc/robot/Robot.java

```diff
@@ -78,6 +80,8 @@ public void robotPeriodic() {
         // commands, running already-scheduled commands, removing finished or interrupted commands,
         // and running subsystem periodic() methods.  This must be called from the robot's periodic
         // block in order for anything in the Command-based framework to work.
+
+        SmartDashboard.putNumber("AnalogSelector1", RobotContainer.analogSelectorOne.getIndex());
```

Comment:
Is this test code?

Reply:
Yes, this was used to test if the selector switch on the robot was working and set to 1.
Do you want me to remove this code?

Reply:
Yes, if its test code we should remove it from the production level code unless we feel it will be used during competition